docs/configuration/indexes.md

```diff
@@ -207,7 +207,9 @@ authenticate = "always"
 ```
 
 When `authenticate` is set to `always`, uv will eagerly search for credentials and error if
-credentials cannot be found.
+credentials cannot be found. Since this is scoped to a specific index, if authentication fails, uv
+will still continue consulting the other indexes you have defined (as well as PyPI, if you have not
```

Comment:
Does it always? Or is this only with the `unsafe-best-match` index strategy? Do we fallback to other indexes on failed authentication with the default strategy?

Reply:
It does in my experiments. But I can explore further before merging this

Reply:
In the registry client, [we call](https://github.com/astral-sh/uv/blob/ab3bab14217d75c06c5d86e2ef1ba65c48091b23/crates/uv-client/src/registry_client.rs#L340) `simple_single_index` with the first index strategy. And in that method [we treat both 401s and 403s](https://github.com/astral-sh/uv/blob/ab3bab14217d75c06c5d86e2ef1ba65c48091b23/crates/uv-client/src/registry_client.rs#L500) as equivalent to "package not found in index" (which is what returning `Ok(None)` signals there).